twitter_acquier.rb

```diff
@@ -0,0 +1,47 @@
+class TwitterAcquier
```

Comment:
I should give better name to them.
But I pushed anyway.

Reply:
`Acquier` is typo.

Reply:
[IMO] Should we separate process for spreadsheet from the class 'TwitterAcquier`.

Reply:
[IMO] Should we separate process for spreadsheet from the class 'TwitterAcquier`.

Reply:
OK! I seperate it!